docs/source/conf.py

```diff
@@ -77,11 +77,36 @@
 
 # Options for HTML output
 # -----------------------
+# Use enthought-sphinx-theme if available
+try:
+    import enthought_sphinx_theme
 
-# The style sheet to use for HTML and HTML Help pages. A file of that name
-# must exist either in Sphinx' static/ path, or in one of the custom paths
-# given in html_static_path.
-html_style = 'default.css'
+    html_theme_path = [enthought_sphinx_theme.theme_path]
+    html_theme = "enthought"
+except ImportError as exc:
+    import warnings
+
+    msg = """Can't find Enthought Sphinx Theme, using default.
+                Exception was: {}
+                Enthought Sphinx Theme can be downloaded from
+                https://github.com/enthought/enthought-sphinx-theme"""
+    warnings.warn(RuntimeWarning(msg.format(exc)))
+
+    # Use old defaults if enthought-sphinx-theme not available
+
+    # The name of an image file (within the static path) to place at the top
+    # of the sidebar.
+    html_logo = "e-logo-rev.png"
```

Comment:
This was `"_static/e-logo-rev.png"` before. Why did it change?

Reply:
Because I copy/pasted from a different repo.  I will fix.

Reply:
Actually, it looks like the `_static` shouldn't be needed any more - it searches inside the `_static` directory by default.

Reply:
It doesn't, unfortunately. The path is relative to the directory containing the configuration file, so if `e-logo-rev.png` is in `_static`, then the `_static` prefix is needed.

Doc: http://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html_logo